source/game/system/KPadDirector.cc

```diff
@@ -11,6 +11,7 @@ void KPadDirector::calc() {
 /// @addr{0x805237E8}
 void KPadDirector::calcPads() {
     m_ghostController->calc();
+    m_hostController->calc();
```

Comment:
Intent here is that, if there is a ghost and a host, the host will take priority? Doesn't look like any inputs from ghost controller would bleed through, so I think this is fine.